### PR TITLE
Feat/dao proposals round labels

### PR DESCRIPTION
**Pull request by Zuckerwattederivat**

### CODE REVIEW

File: components/common/Label.vue [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+export default {
+  name: 'Label',
+
+  components: {},
+
+  props: {},
+
+  data() {
+    return {};
+  },
```
Comment (KinimodEtlon) [inline components/common/Label.vue:29]: - [ ] can be removed if we do not use them

File: components/app/dao-proposals/DaoProposalsList.vue [hunk 2]
```diff
@@ -34,6 +34,11 @@
               </p>
             </div>
           </div>
+          <Label class="mt-5">
+            <span class="text-third"
+              >Round {{ daoProposal.fundingRound.round }}</span
```
Comment (marhali) [inline components/app/dao-proposals/DaoProposalsList.vue:39]: nitpick: please use our existing round translation

See: 
`{{
          $t('project.proposal.round.numbered', {
            round: newestProposal.fundingRound.round,
          })
        }}`

File: components/app/dao-proposals/DaoProposalsList.vue [hunk 3]
```diff
@@ -96,14 +101,16 @@
 
 <script>
 import { CategoryMap } from '@/components/constants/CategoryMap.constant';
-import AppLogo from '~/components/common/AppLogo';
+import AppLogo from '~/components/common/AppLogo.vue';
+import Label from '~/components/common/Label.vue';
```
Comment (marhali) [inline components/app/dao-proposals/DaoProposalsList.vue:105]: chore: we have decided to only use '@' as our path resolver.

File: components/app/dao-proposals/DaoProposalsList.vue [hunk 4]
```diff
@@ -34,6 +34,11 @@
               </p>
             </div>
           </div>
+          <Label class="mt-5">
```
Comment (marhali) [inline components/app/dao-proposals/DaoProposalsList.vue:37]: chore: use vue's best practices regarding naming conventions

see https://vuejs.org/v2/style-guide/#Multi-word-component-names-essential

File: components/common/Label.vue [hunk 5]
```diff
@@ -0,0 +1,31 @@
+<template>
```
Comment (marhali) [inline components/common/Label.vue:1]: issue: the name label is already preserved by the html label tag

Comment (marhali) [reply components/common/Label.vue:1]: suggestion: you can use our own convention for common components like AppLabel

**Review by marhali — CHANGES_REQUESTED** [review]

**Review by Abrom8 — APPROVED** [review]

**Review by marhali — APPROVED** [review]

**Pull request merged**